QSB/TransformSync/PlayerProbeSync.cs

```diff
@@ -45,9 +45,24 @@ protected override Transform InitRemoteTransform()
             return body;
         }
 
-        protected override bool Override()
+        protected override void UpdateTransform()
         {
-            return !Player.GetState(State.ProbeActive);
+            base.UpdateTransform();
+            if (Player.GetState(State.ProbeActive))
+            {
+                return;
+            }
+            if (hasAuthority)
+            {
+                transform.position = ReferenceTransform.InverseTransformPoint(Player.ProbeLauncher.transform.position);
```

Comment:
would be better as `Player.ProbeLauncher.ToolGameObject.transform.position`, like how you did it

Reply:
cool but i consider that outside of ths scope of this PR, you do that after merging ;)
feel free to just merge it btw, don't wait for me :)